### PR TITLE
Doc link to MDE system requirements.  Prevent drift and confusion.

### PR DESCRIPTION
**Pull request by zerotrustfan**
doc link to MDE systems reqs to help protect against drift that creates user confusion

### CODE REVIEW

**Comment by traya1** [comment]
Learn Build status updates of commit _[70ab580](https://github.com/zerotrustfan/memdocs/commits/70ab58074848db06c8ae0b95917ddbc9ca24461f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/intune/protect/advanced-threat-protection.md](https://github.com/zerotrustfan/memdocs/blob/patch-1/memdocs/intune/protect/advanced-threat-protection.md) | :warning:Warning | | [Details](#user-content-49a7851200a71df803189cd76ed2c066-a2e95f0e)

<a id="49a7851200a71df803189cd76ed2c066-a2e95f0e"></a>
### [memdocs/intune/protect/advanced-threat-protection.md](https://github.com/zerotrustfan/memdocs/blob/patch-1/memdocs/intune/protect/advanced-threat-protection.md)
 - **Line 36, Column 89**: **[Warning: hard-coded-locale - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 36, Column 89**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 36, Column 89**: **[Suggestion: preserve-view-not-set - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/preserve-view-not-set?branch=main)]** `````You've pinned this link to a specific version of content with the view parameter. It's recommended not to pin a version unless that version is A) not the default view and B) the context is about that version specifically. To proceed with pinning a version add the &preserve-view=true to the URL.  Otherwise, remove the view parameter. URL: https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212291638162103-3480/BuildReport?accessString=654ec6a0edc94fa8e0edc13541ccacd97fa935d44534e1479f18acfdb9e611a4).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

File: memdocs/intune/protect/advanced-threat-protection.md [hunk 1]
```diff
@@ -33,11 +33,8 @@ ms.collection:
 
 You can integrate Microsoft Defender for Endpoint with Microsoft Intune as a Mobile Threat Defense solution. Integration can help you prevent security breaches and limit the impact of breaches within an organization.
 
-Microsoft Defender for Endpoint works with devices that run:
-- Android
-- iOS/iPadOS
-- Windows 10
-- Windows 11
+Go here to check system requirements for Microsoft Defender for Endpoint:  [Click Here](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide "Microsoft Defender for Endpoint System Requirements"
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/advanced-threat-protection.md:36]:
```suggestion
For the system requirements for Microsoft Defender for Endpoint, see [Minimum requirements for Microsoft Defender for Endpoint](/microsoft-365/security/defender-endpoint/minimum-requirements).
```

File: memdocs/intune/protect/advanced-threat-protection.md [hunk 2]
```diff
@@ -33,11 +33,8 @@ ms.collection:
...
+
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/advanced-threat-protection.md:37]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @zerotrustfan, kindly commit the suggested changes. Many thanks.

**Comment by traya1** [comment]
Learn Build status updates of commit _[d4439d1](https://github.com/zerotrustfan/memdocs/commits/d4439d1d3982560c071e7c7e5146f296a4df5d61)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/intune/protect/advanced-threat-protection.md](https://github.com/zerotrustfan/memdocs/blob/patch-1/memdocs/intune/protect/advanced-threat-protection.md) | :warning:Warning | | [Details](#user-content-49a7851200a71df803189cd76ed2c066-d9f8a55a)

<a id="49a7851200a71df803189cd76ed2c066-d9f8a55a"></a>
### [memdocs/intune/protect/advanced-threat-protection.md](https://github.com/zerotrustfan/memdocs/blob/patch-1/memdocs/intune/protect/advanced-threat-protection.md)
 - **Line 36, Column 89**: **[Warning: hard-coded-locale - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 36, Column 89**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 36, Column 89**: **[Suggestion: preserve-view-not-set - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/preserve-view-not-set?branch=main)]** `````You've pinned this link to a specific version of content with the view parameter. It's recommended not to pin a version unless that version is A) not the default view and B) the context is about that version specifically. To proceed with pinning a version add the &preserve-view=true to the URL.  Otherwise, remove the view parameter. URL: https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212301654427415-3480/BuildReport?accessString=adb0de618c9690f851f0e43dcfb13936a19f002bb42baf2922a6f1b6195b0cc5).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by traya1** [comment]
Learn Build status updates of commit _[05d9bb0](https://github.com/zerotrustfan/memdocs/commits/05d9bb05c04d90c11b53b20f729e5db31f121222)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/intune/protect/advanced-threat-protection.md](https://github.com/zerotrustfan/memdocs/blob/patch-1/memdocs/intune/protect/advanced-threat-protection.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212301701569291-3480/BuildReport?accessString=f57fba0fc1f4f32e64b6a19b2135e4e7494cd767c029fcd6c2f40d80862af42a).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by Brenduns** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**